resources/build/help-keyman-com.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
+
+#
+# Prevents 'clear' on exit of mingw64 bash shell
+#
+SHLVL=0
+
+set -e
+set -u
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+#
+# Allows us to check for existence of subfolders in help/
+#
+shopt -s nullglob
+
+#
+# In this script we update the git repo and copy the Keyman for [Platform]
+# documentation over. This should be run after a full build.
+#
+
+#
+# These are passed via environment:
+#
+# HELP_KEYMAN_COM = the home of the help.keyman.com repository
+# PLATFORM = platform of the Keyman help to upload
```

Comment:
While `HELP_KEYMAN_COM` can be environment, I think `PLATFORM` should be a parameter to the script.